test/supply-test.ts

```diff
@@ -702,4 +662,37 @@ describe('supplyFrom', function () {
     await wait(cometAsB.allow(charlie.address, true));
     await expect(cometAsC.supplyFrom(bob.address, alice.address, COMP.address, 7)).to.be.revertedWith("custom error 'Paused()'");
   });
+
+  it('block re-entrancy', async () => {
```

Comment:
This test seems to be an exact copy of the other reentrancy test.

Reply:
I think before it was removed as there were two tests one test to having exceed supply cap and one test to ensure the inflated accounting number.  Since now both tests will now be reverted right away. 
I think we can go ahead and remove one of them. Since both are identical and both got reverted right away.